### DISCUSSION TITLE
Disabled User can still use the website

### DISCUSSION THREAD

**Discussion by justsWn** [OrchardCMS/OrchardCore#15717] [Q&A] [answered]
When using the Orchard Core default login: When I have a logged in user which gets disabled by an admin, the user does not lose access to the system. When he logs out and tries to log in again, the login fails with a correct message. But I want the user to lose access immediately.
Is this intended behavior and due to the underlying technical construct?
Is there a possibility that the user is logged out?
It would be sufficient for me if removing the role would have an immediate effect, but the user can still change the role of other users after removing his permission for that.

**Comment by lampersky** [comment, marked as answer]
@justsWn By default security stamps have 30 minutes validation interval.
You can configure it to 0 minutes. This way, once a user is disabled (or role is edited), he or she will lose access within the next request.

```
services.Configure<SecurityStampValidatorOptions>(options =>
{
    options.ValidationInterval = TimeSpan.Zero;
});
```

**Reply by justsWn** [reply]
Works perfectly! THANK YOU so much. Somehow I didn't think about the standard ASP.NET mechanism (and I didn't know about this one either..)
